android/kotlin-central-login-oidc/app/src/main/java/com/example/app/Destinations.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2023 - 2024 ForgeRock. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+package com.example.app
+
+object Destinations {
+    const val ENV_ROUTE = "Environment"
+    const val TOKEN_ROUTE = "Access Token"
+    const val DEVICE_PROFILE = "Device Profile"
```

Comment:
Still not sure if we want "Device Profile" functionality for this app... Let's confirm this...